pkg/experimental/how-to.md

```diff
@@ -0,0 +1,63 @@
+## The Manual Way
+
+- add a blank import of module inside `cmd/golangci-lint/plugins.go`
```

Comment:
It's worth mentioning here that this is a new file and doesn't need to be called exactly like that. When I first read this, it looked like it already exists and needs to be extended.

Reply:
This file already exists but only in my PR :wink: